pkg/githubclient/client_repository.go

```diff
@@ -247,15 +248,34 @@ func (c *Client) getGithubChecks(ctx context.Context, repository *github.Reposit
 	var checks []string
 	for _, combinedStatus := range allCombinedStatus {
 		for _, status := range combinedStatus.Statuses {
-			checks = append(checks, *status.Context)
+			if filterCheck(status.GetContext()) {
+				checks = append(checks, status.GetContext())
+			}
 		}
 	}
 
 	c.logger.Debugf("found %d commit statuses for ref %q:\n%v", len(checks), ref, checks)
+	for id, check := range checks {
+		c.logger.Debugf(" - checks[%d] = %q", id, check)
+	}
 
 	return checks, nil
 }
 
+// Returns false if check should be ignored
+func filterCheck(checkName string) bool {
+	// List keywords for checks that should be ignored
+	var ignoredCheckKeywords = []string{"aliyun"}
+
+	for _, ignoredCheckKeyword := range ignoredCheckKeywords {
+		matched, _ := regexp.MatchString(ignoredCheckKeyword, checkName)
```

Comment:
Let's not ignore the returned error, this is bad practice.
We can also compile the regex beforehand (I'm thinking when we get the flag value), and just use `MatchString(string) bool` method here.